build.gradle

```diff
@@ -296,6 +296,7 @@ task commonDistRuntime(type: Copy) {
 task crossDistRuntime(type: Copy) {
     dependsOn.addAll(targetList.collect { "${it}CrossDistRuntime" })
     dependsOn('commonDistRuntime')
+    dependsOn(':backend.native:teamcityPublishStdLibSources')
```

Comment:
May it be better to make stdlib build task be `finalizedBy` the zip task that will print the desired line in its `doLast` part?
And wherever the stdlib build is triggered it will publish the sources, while top level tasks could be changed independently without the need to add such dependence.